_includes/package_links.html

```diff
@@ -41,16 +41,6 @@
       {% assign is_ros1_page = true %}
     {% endif %}
   {% endfor %}
-  
-  {% if is_ros1_page %}
-  <a class="{{list_group_class}} {% comment %} TODO: rosindex currently does not detect if a wiki page exists - implement and then re-enable the following: {% unless package.data.wiki.exists %}inactive{% endunless %} {% endcomment %}"
-     target="_blank"
-     href="http://wiki.ros.org/{{page.package_name}}?distro={{distro}}"
-     title="View this package on wiki.ros.org">
-       <span style="margin-right: 5px;" class="glyphicon glyphicon-info-sign"></span>
-       {%if hide_link_labels%}<br>{%endif%}
-       Wiki
-  </a>
-  {%endif%}
+
```

Comment:
is_ros1_page is only used in the code you removed below, so also remove the generation of it, which is in the code above.